common/src/thrift/common.thrift

```diff
@@ -44,6 +44,12 @@ struct NetAddress {
   3: i32 dataPort
 }
 
+// Information about the RPC.
+struct RpcOptions {
+  // key used to identify retried RPCs
+  1: optional string key
```

Comment:
I prefer id; it just happens that the use right now is as a key in the cache. I've created a JIRA to track this discussion: https://tachyon.atlassian.net/browse/TACHYON-1308